src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -86,6 +247,10 @@ export default class ReplyField extends React.Component<Props, State> {
                     readOnly={isDisabled}
                     stripPastedStyles
                 />
+
+                {popupReference && (
+                    <PopupList modifiers={this.popupModifiers} reference={popupReference as HTMLElement} />
```

Comment:
This cast shouldn't be needed.

Reply:
> This cast shouldn't be needed.

`popupReference` has type `VirtualElement` which is imported from popper module, and `VirtualElement` is missing some properties from `HTMLElement`. So it has to be casted.

Reply:
Addressed here: https://github.com/box/box-annotations/pull/470#discussion_r419625261